blog/_posts/2024-02-26-userfeedback-2024.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: blog_post
+title: "Informed Transformation: User Feedback’s Role in the ADS to SciX Transition"
+author: Brit Myers
+position: SciX Community Engagement Consultant
+category: blog
+label: news
+thumbnail: blog/images/blog_2024-02-26_usersurvey.png
```

Comment:
Check the file name - if this is the right file, it wasn't included in the PR. (This also applies to the file in `scixblog/_posts`

Reply:
changed thumbnail file name to blog_2024-02-26_surveyresponse_1.png (first image in the blog & included with image files)